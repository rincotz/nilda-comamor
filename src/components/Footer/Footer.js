import React from 'react';
import { Link } from 'react-router-dom';

const Footer = () => (
    <nav>
        <header><Link to='/'>.comAmor</Link></header>
        <ul>
            <li><Link to='/sobre'>Quem somos</Link></li>
            <li><Link to='/carreiras'>Carreiras</Link></li>
            <li><Link to='/ajuda'>Ajuda</Link></li>
        </ul>
        <ul>
            <li><Link to='/cozinhando'>Cozinhando</Link></li>
            <li><Link to='/cadastro'>Cadastro</Link></li>
            <li><Link to='/cozinhando/porque'>Por que cozinhar</Link></li>
            <li><Link to='/cozinhando/hospitalidade'>Hospitalidade</Link></li>
            <li><Link to='/cozinhando/responsabilidade'>Cozinhando com responsabilidade</Link></li>
        </ul>
    </nav>
)

export default Footer;