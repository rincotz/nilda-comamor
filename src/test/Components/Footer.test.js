import React from 'react'
import { shallow } from 'enzyme'
import Footer from '../../components/Footer/Footer'

test('should render Header correctly', () => {
    const wrapper = shallow(<Footer />)
    expect(wrapper).toMatchSnapshot()
})