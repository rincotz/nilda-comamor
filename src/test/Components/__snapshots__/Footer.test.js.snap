// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<nav>
  <header>
    <Link
      replace={false}
      to="/"
    >
      .comAmor
    </Link>
  </header>
  <ul>
    <li>
      <Link
        replace={false}
        to="/sobre"
      >
        Quem somos
      </Link>
    </li>
    <li>
      <Link
        replace={false}
        to="/carreiras"
      >
        Carreiras
      </Link>
    </li>
    <li>
      <Link
        replace={false}
        to="/ajuda"
      >
        Ajuda
      </Link>
    </li>
  </ul>
  <ul>
    <li>
      <Link
        replace={false}
        to="/cozinhando"
      >
        Cozinhando
      </Link>
    </li>
    <li>
      <Link
        replace={false}
        to="/cadastro"
      >
        Cadastro
      </Link>
    </li>
    <li>
      <Link
        replace={false}
        to="/cozinhando/porque"
      >
        Por que cozinhar
      </Link>
    </li>
    <li>
      <Link
        replace={false}
        to="/cozinhando/hospitalidade"
      >
        Hospitalidade
      </Link>
    </li>
    <li>
      <Link
        replace={false}
        to="/cozinhando/responsabilidade"
      >
        Cozinhando com responsabilidade
      </Link>
    </li>
  </ul>
</nav>
`;
